src/object_functions.c

```diff
@@ -578,11 +576,11 @@ void obj_loop_trophycab(Object *obj, s32 speed) {
         if ((raceType != 7) && (raceType != 6)) { // Make sure the current level is not a cutscene.
             obj->unk7C.word = 1;
             if ((settings->trophies >> (((settings->worldId - 1) ^ 0) * 2)) & 3) {
-                sp44.unk0 = 128;
-                sp44.unk2 = obj->segment.unk3C_a.unk3C->unk2;
-                sp44.unk4 = obj->segment.unk3C_a.unk3C->unk4;
-                sp44.unk6 = obj->segment.unk3C_a.unk3C->unk6;
-                sp44.unk1 = 8;
+                sp44.objectID = 128;
+                sp44.yRotation = obj->segment.unk3C_a.unk3C->unk2;
+                sp44.xRotation = obj->segment.unk3C_a.unk3C->unk4;
+                sp44.zRotation = obj->segment.unk3C_a.unk3C->unk6;
+                sp44.size = 8;
```

Comment:
This looks like a level object entry struct to me. Maybe look at `LevelObjectEntryCommon` in `include/level_object_entries.h`?